cli/azd/cmd/testdata/TestUsage-azd.snap

```diff
@@ -2,21 +2,21 @@ Usage:
   azd [command]
 
 Available Commands:
-  config      Manage Azure Developer CLI configuration
-  deploy      Deploy the application's code to Azure.
-  down        Delete Azure resources for an application.
-  env         Manage environments.
-  infra       Manage Azure resources.
-  init        Initialize a new application.
-  login       Log in to Azure.
-  logout      Log out of Azure
-  monitor     Monitor a deployed application.
-  pipeline    Manage GitHub Actions pipelines.
-  provision   Provision the Azure resources for an application.
-  restore     Restore application dependencies.
-  template    Manage templates.
-  up          Initialize application, provision Azure resources, and deploy your project with a single command.
-  version     Print the version number of Azure Developer CLI.
+  config           Manage Azure Developer CLI configuration
```

Comment:
TIL that even if the command is hidden, Cobra still uses its length when figuring out how to layout this table.  That's why we see the whitespace diff here.

I think there's a question as to if this command should be hidden or not.  It felt a little strange to see it at top level (all the other commands at this level are short are more or less friendly words and then having `get-access-token` sitting there felt really weird, so I kept it hidden for now.

In `az` this is called `az account get-access-token`.  I am wondering if an account (or perhaps an `auth` prefix?) would make sense here.

I could imagine if we introduce `account` or `auth` we could put `login` and `logout` under there as well (in addition to aliases at top level, to make these two actions more discoverable).

Thoughts @savannahostrowski?

Reply:
![image](https://user-images.githubusercontent.com/24213737/202546174-1361531a-d2a5-4e5d-aa9b-48a25ea5b31c.png)

Reply:
We also need a way to check if azd is logged in or not.  like `azd auth show`

Reply:
Hm, I'm a little worried we'd be moving away from one of the central tenets of azd (to be intentionally high level) by exposing this as a public-facing command, especially because we don't currently have this as an ask from our users. I'd still prefer to keep this private until we have a compelling user case. This seems a little more granular than other commands we currently expose.

Reply:
> We also need a way to check if azd is logged in or not. like azd auth show

FWWIW, we have this today via `azd login --check-status` (which VS uses) but I would not be opposed to adding another entry point.

Reply:
I changed the name here to `azd auth token`.  `azd auth` is hidden for now.